lib/collection/src/operations/point_ops.rs

```diff
@@ -246,6 +246,16 @@ impl Validate for Batch {
                     }
                 }
             }
+            BatchVectorStruct::Sparse(named_vectors) => {
```

Comment:
This looks a bit ad-hoc.
Why is it gain that we can't evolve `BatchVectorStruct` to use `Vector` in place of `VectorType`?
Like it is done for `VectorStruct`.

Reply:
fixed by removing `BatchVectorStruct::Sparse`